packages/divider/package.json

```diff
@@ -41,18 +40,15 @@
     ],
     "sideEffects": [
         "./sp-*.js",
-        "./sp-*.ts",
-        "./stories/typography-decorator.js",
-        "./stories/typography-decorator.ts"
```

Comment:
These are files that we do need to list as side effects in order for Storybook to load this file correctly and display [the new visual regression](https://circle-production-customer-artifacts.s3.amazonaws.com/picard/5bf34b122c07f700013ff527/60108f5441417e2e36e7a748-0-build/artifacts/test/visual/screenshots-current/ci/divider--large__dark__large__ltr.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20210126T224453Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20210126%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=fbd4e9384c717dc7263cfce4f68c0f9af12f733e00afad30a9dc72d60a1086d4) and [the old one](https://circle-production-customer-artifacts.s3.amazonaws.com/picard/5bf34b122c07f700013ff527/6010837243451e697ac4d8e5-0-build/artifacts/test/visual/screenshots-current/ci/rule--large__dark__large__ltr.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20210126T224523Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20210126%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=37233611fa676599d8fecebfe0db11e178076cf857970a8175685c7f3c181977) the same.